easybuild/easyconfigs/x/xorg-macros/xorg-macros-1.19.1-GCCcore-6.3.0.eb

```diff
@@ -1,15 +1,26 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     Custom
+#
+# Notes::
+##
+
 easyblock = 'ConfigureMake'
 
 name = 'xorg-macros'
 version = '1.19.1'
 
-homepage = 'http://cgit.freedesktop.org/xorg/util/macros'
+homepage = 'https://cgit.freedesktop.org/xorg/util/macros'
 description = """X.org macros utilities."""
 
 toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
 
-source_urls = ['http://cgit.freedesktop.org/xorg/util/macros/snapshot']  # no slash ('/') at the end!
+source_urls = ['https://gitlab.freedesktop.org/xorg/util/macros/-/archive/util-macros-%(version)s']
 sources = ['util-macros-%(version)s.tar.gz']
```

Comment:
@robqiao You're using `util-macros-1.19.1.tar.gz` here, while you use `macros-util-macros-1.19.1.tar.gz` in `xorg-macros-1.19.1-GCCcore-6.4.0.eb`, is that intentional?

The source code is identical though if you check the unpacked directories (just the top-level dir is different):

```
$ tar xfvz util-macros-1.19.1.tar.gz
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/.gitignore
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/COPYING
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/INSTALL
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/Makefile.am
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/README
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/autogen.sh
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/configure.ac
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/xorg-macros.m4.in
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/xorg-macros.pc.in
x macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5/xorgversion.m4

$ tar xfvz macros-util-macros-1.19.1.tar.gz
x macros-util-macros-1.19.1/
x macros-util-macros-1.19.1/.gitignore
x macros-util-macros-1.19.1/COPYING
x macros-util-macros-1.19.1/INSTALL
x macros-util-macros-1.19.1/Makefile.am
x macros-util-macros-1.19.1/README
x macros-util-macros-1.19.1/autogen.sh
x macros-util-macros-1.19.1/configure.ac
x macros-util-macros-1.19.1/xorg-macros.m4.in
x macros-util-macros-1.19.1/xorg-macros.pc.in
x macros-util-macros-1.19.1/xorgversion.m4

$ diff -ru macros-util-macros-1.19.1 macros-util-macros-1.19.1-4b6b1dfea16214b5104b5373341dc8bc7016d0b5
$ 
```

So, I would opt for using `macros-util-macros-%(version)s.tar.gz` everywhere, partially because that helps to avoid that people will run into checksum failures if they have the previous source tarballs already downloaded.

The good news is that the contents of the (previous) `util-macros-1.19.1.tar.gz` and `macros-util-macros-1.19.1.tar.gz` is identical:

```
$ diff -ru macros-util-macros-1.19.1 util-macros-1.19.1
$ 
```